db/migrate/20170817064448_create_tenant_api_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTenantApiRequests < ActiveRecord::Migration
+  def change
+    create_table :tenant_api_requests do |t|
+      t.integer :tenant_id, null: false
+      t.integer :count, default: 0
```

Comment:
It appears that a `TenantApiRequest` tracks the number of requests by a Tenant for a single day. That makes sense, but I think the class's name is misleading in that case.